storage/innobase/handler/ha_innodb.h

```diff
@@ -409,6 +411,10 @@ class ha_innobase : public handler {
                                   bool commit, const dd::Table *old_dd_tab,
                                   dd::Table *new_dd_tab);
   /** @} */
+  inline MY_ATTRIBUTE((warn_unused_result)) bool commit_try_rebuild(
```

Comment:
No definition available  - so not inline
```suggestion
  MY_ATTRIBUTE((warn_unused_result)) bool commit_try_rebuild(
```